test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java

```diff
@@ -65,6 +77,7 @@ public void run() {
             }
         });
 
+        robot.delay(250);
         robot.waitForIdle();
```

Comment:
Not positive if it matters, but from what I've seen the delay usually comes after waitForIdle.

But I also ran the test and it seems to be OK.